.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+# Vsode editor files
+/.vscode
```

Comment:
Your commit Message is not describing the action done here. What you did was actually ignoring the vscode generated files. Let the message be precise.

Reply:
Well noted.
I squashed the commits to include a more descriptive commit message.

Reply:
Great, resolved.